SlackAPI/SlackClient.cs

```diff
@@ -147,6 +147,22 @@ public void ChannelsInvite(Action<ChannelInviteResponse> callback, string userId
             APIRequestWithToken(callback, parameters.ToArray());
         }
 
+        public void GetConversationsList(Action<ConversationsListResponse> callback, string cursor = "", bool ExcludeArchived = true, int limit = 100, string[] types = null)
+        {
+	        List<Tuple<string, string>> parameters = new List<Tuple<string, string>>()
+	        {
+		        Tuple.Create("exclude_archived", ExcludeArchived ? "1" : "0")
+	        };
+	        if (limit > 0)
+		        Tuple.Create("limit", limit.ToString()); 
```

Comment:
Just realized these aren't actually getting passed into the list of parameters

Reply:
Silly mistake! Just updated this with the correct code.